File: src/telegram/types/shipping-address.ts
import * as type from '../types'
import Type from './type'


export declare type JsonShippingAddress = {
	country_code: type.String
	state: type.String
	city: type.String
	street_line1: type.String
	street_line2: type.String
	post_code: type.String
}


export default class ShippingAddress extends Type {
	country_code: type.String
	state: type.String
	city: type.String
	street_line1: type.String
	street_line2: type.String
	post_code: type.String

	defaultObject!: JsonShippingAddress

	constructor(object: JsonShippingAddress) {
		super()

		this.defaultObject = object

		this.country_code = object.country_code
		this.state = object.state
		this.city = object.city
		this.street_line1 = object.street_line1
		this.street_line2 = object.street_line2
		this.post_code = object.post_code
	}
}

File: telegram-api-json/pygen/ShippingAddress.py

from telelib.bot import DefaultType

# @url https://core.telegram.org/bots/api#shippingaddress# ['This object represents a shipping address.']
class ShippingAddress(DefaultType):

    
	# @var country_code Two-letter ISO 3166-1 alpha-2 country code
	@property
	def country_code(self) -> String:
		return self._d["country_code"]
	# @var state State, if applicable
	@property
	def state(self) -> String:
		return self._d["state"]
	# @var city City
	@property
	def city(self) -> String:
		return self._d["city"]
	# @var street_line1 First line for the address
	@property
	def street_line1(self) -> String:
		return self._d["street_line1"]
	# @var street_line2 Second line for the address
	@property
	def street_line2(self) -> String:
		return self._d["street_line2"]
	# @var post_code Address post code
	@property
	def post_code(self) -> String:
		return self._d["post_code"]

